File: web-dev/cs50/intro-comp-science/Problem-sets/stuff/mario.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>

void draw(int x);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height of you pyramid: ");
    }
    while (height < 1 || height > 8)

    darw(height);
}

void draw(int x)
{
    if(x == 0)
    {
        return;
    }
    draw(h - 1);

    // Drawing the row of ##
    for (i = 0; i < x; i++)
    {
        printf("#");
    }
    printf("\n");

}

File: web-dev/cs50/intro-comp-science/Problem-sets/pset6/mario_more/mario.py
from cs50 import get_int

## Starting with our main program
def main():
    while True:
        ## Getting an integer input from the user defined as the height
        height = get_int("Height: ")
        width = height
        ## Setting a condition that the user's input must fulfill
        if height > 0 and height <= 8:
            break
    ## Calling our function that will draw the pyramid
    for i in range(1, height + 1):
        num_hashes = i
        num_spaces = width - i
        print(" " * num_spaces, end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)

if __name__ == "__main__":
    main()

